framework/wazuh/core/cluster/cluster.py

```diff
@@ -98,27 +157,61 @@ def check_cluster_status():
 
 def walk_dir(dirname, recursive, files, excluded_files, excluded_extensions, get_cluster_item_key, get_md5=True,
              whoami='master'):
+    """Recursively iterate inside a directory, save the path of each found file and obtain its metadata.
+
+    Parameters
+    ----------
+    dirname : str
+        Directory within which to look for files.
+    recursive : bool
+        Whether to recursively look for files inside found directories.
+    files : list
+        List of files to obtain information from.
+    excluded_files : list
+        List of files to ignore.
+    excluded_extensions : list
+        List of extensions to ignore.
+    get_cluster_item_key : str
+        Key inside cluster.json['files'] to which each file belongs. This is useful to know what actions to take
+        after sending a file from one node to another, depending on the directory the file belongs to.
+    get_md5 : bool
+        Whether to calculate and save the MD5 hash of the found file.
+    whoami : str
+        TODO - To be deprecated.
```

Comment:
???

Reply:
The parameter will be deleted in other issue (https://github.com/wazuh/wazuh/issues/6475).